packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -108,14 +109,22 @@ export class SDKClient {
         this.operatorAccountId = clientMain.operatorAccountId ? clientMain.operatorAccountId.toString() : 'UNKNOWN';
 
         // clear and create metrics in registry
-        const metricHistogramName = 'rpc_relay_consensusnode_response';
-        register.removeSingleMetric(metricHistogramName);
-        this.consensusNodeClientHistorgram = new Histogram({
-            name: metricHistogramName,
+        const metricHistogramCost = 'rpc_relay_consensusnode_cost';
+        register.removeSingleMetric(metricHistogramCost);
+        this.consensusNodeClientHistorgramCost = new Histogram({
+            name: metricHistogramCost,
             help: 'Relay consensusnode mode type status cost histogram',
             labelNames: ['mode', 'type', 'status', 'caller', 'interactingEntity'],
             registers: [register]
         });
+        const metricHistogramGasFee = 'rpc_relay_consensusnode_gasfee';
+        register.removeSingleMetric(metricHistogramGasFee);
+        this.consensusNodeClientHistorgramGasFee = new Histogram({
+            name: metricHistogramGasFee,
+            help: 'Relay consensusnode mode type status gas fee histogram',
```

Comment:
same as above, the Help field, is a text description that helps the consumer of the metrics to understand what this metric is being used for, I would change it for something like:
"Relay consensus node histogram for capturing amount of gas fees charged by mode, type, status, caller and interactingEntity"

Reply:
Agreed.  A more descriptive hint in the help if the text can fit, is preferable.